packages/tools/viewer/src/viewer.ts

```diff
@@ -861,7 +862,7 @@ export class Viewer implements IDisposable {
             },
             suspendRendering: () => this._suspendRendering(),
             markSceneMutated: () => this._markSceneMutated(),
-            pick: (screenX: number, screenY: number) => this._pick(screenX, screenY),
+            pick: async (screenX: number, screenY: number) => this._pick(screenX, screenY),

```

Comment:
Why is `async` needed here?

Reply:
Discussed with Gary previously. Since it returns a promise, a flag (that is now disabled) marked it as potential for an async function. There is semantically and technically no different between this function with async or without async as it does nothing more than return a promise.
I didn't add async to functions that run some code before returning a promise, as this would be a breaking change.